ci/test_wheel_cuopt.sh

```diff
@@ -59,3 +59,4 @@ timeout 10m bash ./python/libcuopt/libcuopt/tests/test_cli.sh
 # Run Python tests
 RAPIDS_DATASET_ROOT_DIR=./datasets timeout 30m python -m pytest --verbose --capture=no ./python/cuopt/cuopt/tests/
 
+timeout 3m ./ci/external/cvxpy_smoketest.sh
```

Comment:
```suggestion
timeout 3m ./ci/external/cvxpy_smoketest.sh
```

3 minutes is a REALLY tight timeout for a script that includes network calls over the internet like cloning a git repo and installing packages from `pypi.org`.

I recommend either increasing this to something like `30m` or, if the goal is just to ensure that test code runs quickly, moving the `timeout` down into `cvxpy_smoketest.sh`.

Reply:
I should also mention, in the description you wrote:

> It does not do any performance testing or building.

Putting such a restrictive timeout on the test run is a form of "performance testing". `cuopt` or `cvxpy` getting much slower would cause that timeout to be exceeded and the job to fail. So I think that statement in the PR description is not quite correct.

Reply:
And I will also address other issues you pointed out on 3m run-time. I think @tmckayus added 3m time line similar to our other test to kind trigger a warning in case somethings suspiciously slow, or someone added too many tests. I think 3m is a good amount of time for a smoke test in cuOpt.

Reply:
I will update the PR description.

Reply:
I did this as part of these changes: https://github.com/NVIDIA/cuopt/pull/199#discussion_r2206124291

Reply:
Thank you @jameslamb